iron-form.html

```diff
@@ -343,7 +342,9 @@
     },
 
     _registerElement: function(e) {
-      var element = e.target;
+      // Get the actual element that fired the event
+      var element = Polymer.dom(e).path[0];
```

Comment:
`Polymer.dom(e).rootTarget` should be a bit better in terms of performance (since you need only one element and not the whole path tree)